recipe/meta.yaml

```
@@ -1,39 +1,66 @@
 {% set name = "visvis" %}
-{% set version = "1.13.0" %}
-{% set sha256 = "319d2e37c9756a029be33dc491461b8359fc52caa9c5147807eba583cafbf749" %}
+{% set version = "1.15.0" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  sha256: aa013c62ea54ae4d170096e1fced7c3697b1d05c2fe6b38e822af0a986422a39
 
 build:
-  noarch: python
-  script: python -m pip install --no-deps --ignore-installed .
   number: 0
+  skip: true  # [py<27]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed --no-cache-dir -vv
 
 requirements:
   host:
     - python
     - pip
+    - flit >=3.2,<4
```

```suggestion
    - flit-core >=3.2,<4
```

See https://github.com/almarklein/visvis/blob/main/pyproject.toml#L45. `grayskull` also suggests to use this package

---

Yep, but we don't have flit-core package in conda default repo, we have flit - this one include flit-core

---

How so? https://anaconda.org/anaconda/flit-core